README.md

```diff
@@ -35,38 +35,34 @@
 ## Overview
 ![screenshot](https://user-images.githubusercontent.com/3839533/163580222-83029e1f-81a0-419f-9b05-ea8c5571a3c0.png)
 
-
-
-This is to create a small version of Airbnb - Windbnb with given data. It was recomennded that we either use React or Vue. I did this with just plain Javascript.
+This is to create a mock version of "Airbnb", called Windbnb - with dummy data. It was recomennded that we either use React or Vue. I did this with just plain Javascript.
 
 ### Struggles
 - Fetching the data with the fetch API. Faced [this](https://stackoverflow.com/questions/45696999/fetch-unexpected-end-of-input) issue.
    - Left a branch open for this: `Fetch-EOI-issue` [link](https://github.com/pamdayne/wind-d-bnb/tree/Fetch-EOI-issue)
 
-// TODO: Fill in these experiences:
 ### Experiences
 TBA
 
 ### Learned/Improved
-- learnt one thing about grid `repeat` usage `grid-templace-column: repeat(3, 31%)`
-
-### Wisdom
-TBA
+- Learnt about grid `repeat` usage `grid-templace-column: repeat(3, 31%)`
```

Comment:
Any reason why you remove `grid-template-column` and replace with `flex`?

Reply:
I think it was because any elements after a div with `grid-template-column` gets covered by it.
Using flex works well after that.

I do wanna use the grid though. I'll find some other time to learn deeper about it and how to work with it.